modules/ROOT/pages/server_release_notes.adoc

```diff
@@ -162,23 +162,30 @@ Find below a list of updated apps in comparison with the 10.12.2 complete bundle
 
 * https://marketplace.owncloud.com/apps/activity [Activity] 2.7.2
 * https://marketplace.owncloud.com/apps/admin_audit [Auditing] 2.1.4
+* https://marketplace.owncloud.com/apps/configreport [ConfigReport] 0.2.2
 * https://marketplace.owncloud.com/apps/customgroups [Custom Groups] 0.7.2
 * https://marketplace.owncloud.com/apps/diagnostics [Diagnostics] 0.2.0
+* https://marketplace.owncloud.com/apps/drawio [Drawio] 1.0.1
+* https://marketplace.owncloud.com/apps/encryption [Default encryption module] 1.6.1
 * https://marketplace.owncloud.com/apps/files_antivirus [Anti-Virus] 1.2.2
 * https://marketplace.owncloud.com/apps/files_classifier [Document Classification] 1.4.0
 * https://marketplace.owncloud.com/apps/files_lifecycle [File Lifecycle Management] 1.3.4
 * https://marketplace.owncloud.com/apps/files_primary_s3 [S3 Primary Object Storage] 1.5.0
-* https://marketplace.owncloud.com/apps/files_texteditor [Text Editor] 2.5.2
+* https://marketplace.owncloud.com/apps/files_texteditor [Text Editor] 2.6.0
+* https://marketplace.owncloud.com/apps/firstrunwizard [First run wizard] 1.3.0
 * https://marketplace.owncloud.com/apps/impersonate [Impersonate] 0.5.3
+* https://marketplace.owncloud.com/apps/kerberos [Kerberos Integration] 1.0.0
 * https://marketplace.owncloud.com/apps/market [Market] 0.8.0
-* https://marketplace.owncloud.com/apps/metrics [Metrics] 1.3.0
+* https://marketplace.owncloud.com/apps/metrics [Metrics] 1.4.0
+* https://marketplace.owncloud.com/apps/notifications [Notifications] 0.6.0
 * https://marketplace.owncloud.com/apps/oauth2 [OAuth2] 0.6.0
 * https://marketplace.owncloud.com/apps/password_policy [Password Policy] 2.2.0
 * https://marketplace.owncloud.com/apps/ransomware_protection [Ransomware Protection] 1.7.0
+* https://marketplace.owncloud.com/apps/updater [Updater] 1.1.0
```

Comment:
I don't see the updater app in the `owncloud-complete-20230821.tar.bz2`

Is it supposed to be in the bundle?

Reply:
@jnweiger ^

Reply:
Yes, it is there, but it is hiding. This app is so very special, it has no appinfo and it lives in "targetdirectory": "owncloud/updater" instead of "owncloud/apps/..."